### PR TITLE
Add difficulty mask

### PR DESCRIPTION
**Pull request by hx507**
Address issue #34.

### CODE REVIEW

**Comment by hx507** [comment]
Tests are failing with 
```
ERROR: Could not find a version that satisfies the requirement tensorflow>=2.1.0
ERROR: No matching distribution found for tensorflow>=2.1.0
```

**Comment by alvinsunyixiao** [comment]
Good catch. Yeah, that is Python 3.9 and TensorFlow is not yet compatible. The fix is in #45.

File: dnn/data/converters/roco.py [hunk 1]
```diff
@@ -9,6 +9,13 @@
     https://uillinoisedu-my.sharepoint.com/:u:/g/personal/yixiaos3_illinois_edu/Ef5LIWMYRpNLgMuXGHOrAZoBVKdsymnJ8xWinbJO_pX9OQ?e=POF0Ts
 """
 
+import sys
+import os
+sys.path.insert(0, os.path.abspath('.').split('/dnn')[0])
+print('Trim path to',sys.path[0])
```
Comment (alvinsunyixiao) [inline dnn/data/converters/roco.py:15]: This should be done in a virtual environment, by `export PYTHONPATH=<path to irm_Autonomy_2020>:$PYTHONPATH`.

File: dnn/data/converters/roco.py [hunk 2]
```diff
@@ -55,35 +62,50 @@ def convert_annot(annot_path):
...
-                image_target, object_target, bbox_target = convert_annot(annot_path)
+                image_target, object_target, bbox_target, difficulties = convert_annot(
+                    annot_path)
                 example = tf.train.Example(features=tf.train.Features(feature={
                     'image': bytes_feature(image_target),
                     'class_n': bytes_feature(object_target),
                     'bbox_yxyx_n4': bytes_feature(bbox_target),
+                    '''
+                    difficulty:
+                        -1: DNE
```
Comment (alvinsunyixiao) [inline dnn/data/converters/roco.py:103]: Maybe it's worth updating the purdue converter by setting all difficulties to -1. But if we use it only for verification then probably just leave it as it is

**Review by alvinsunyixiao — CHANGES_REQUESTED** [review]
Looks good to me, other than the Python library path should be set in the system but not in the script.

File: dnn/data/converters/roco.py (hunk 2 above)
Comment (D0048) [reply dnn/data/converters/roco.py:98]: Added that.

**Review by alvinsunyixiao — APPROVED** [review]
LGTM
